WooCommerce/Classes/ViewRelated/Notifications/ReviewsCoordinator.swift

```diff
@@ -0,0 +1,110 @@
+import Foundation
+import UIKit
+
+import enum Yosemite.ProductReviewAction
+import enum Yosemite.NotificationAction
+import struct Yosemite.ProductReviewFromNoteParcel
+
+/// Coordinator for the Reviews tab.
+///
+final class ReviewsCoordinator: Coordinator {
+    var navigationController: UINavigationController
```

Comment:
Unfortunately, I can't do this yet because this is from `Coordinator`. 🤷 

https://github.com/woocommerce/woocommerce-ios/blob/13869bcb7b3f7d6c974e45bbee2ff62e2609afa3/WooCommerce/Classes/Tools/Coordinator.swift#L7-L12

Reply:
ooh it's also settable! no problem then 👍